Simplified/Book/Models/NYPLBookRegistry.m

```diff
@@ -454,10 +454,11 @@ - (void)removeExpiredBannedBooks
     NSMutableArray *booksToRemove = [[NSMutableArray alloc] init];
     for (NSString *bookIdentifer in self.identifiersToRecords) {
       NYPLBookRegistryRecord *record = self.identifiersToRecords[bookIdentifer];
+      NYPLOPDSAcquisition *acquisition = record.book.defaultAcquisition;
       // Add the book to remove list if it is distributed by Axis360 and expired
-      if([record.book.defaultAcquisition.type isEqualToString:ContentTypeAxis360]) {
-          if (record.book.defaultAcquisition.availability.until &&
-            [record.book.defaultAcquisition.availability.until compare:[NSDate date]] == NSOrderedAscending) {
+      // or with no expiration
+      if ([acquisition.type isEqualToString:ContentTypeAxis360]) {
+          if ([acquisition.availability.until compare:[NSDate date]] != NSOrderedDescending) {
```

Comment:
@russellcullen  in objc if `acquisition.availability.until` is nil, the function call `[nil compare:[NSDate date]]` will not result in a crash and instead will just return `0`. This is pretty common in objc! 

So, then, `0` is `!= NSOrderedDescending` and test succeeds 🎉